.github/workflows/python-package.yml

```diff
@@ -11,7 +10,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ["3.9"]
```

Comment:
Why not 3.8? Pyaleph is using 3.8 for at least the next few months.

Reply:
Python 3.8 should be supported as long as we support Ubuntu 20.04.
We should first get this thing to work however.